### PR TITLE
Module 5

### PR DESCRIPTION
**Pull request by CreaTiVeUK**
- Completed Module 5 + Stretch goals.
- Bug fix on switching cards to a different list.
- Removed: Commented code, Incomplete tests, Duplicate function.
- No longer returning the lists as part of get_Items_and_Status().

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Creating folders, and files for a project:
+COPY todo_app ./todo_app
+
+# -----Configure for local test-----
+FROM base as test
+ENV FLASK_ENV development
+ENV FLASK_DEBUG 1
+# Define an entrypoint, and default launch command
+ENTRYPOINT poetry run pytest; poetry run watchmedo shell-command \
```
Comment (hugh-emerson) [inline Dockerfile:25]: Iirc, you were on Windows so you saw that the file watching didn't really work. If you're interested in trying this again outside the devops course, [pytest-watch](https://pypi.org/project/pytest-watch/) has a `--poll` option for watching the files for changes. This works around the Windows/Docker inotify event issue.

File: README.md [hunk 2]
```diff
@@ -89,6 +89,46 @@ $ ansible-playbook --ask-vault-pass deploy_todoApp.yaml -i inventory
...
+
+2. Install GUnicorn:
+```bash
+$ pip install gunicorn
+```
+
+3. Install Watchdog
+```bash
+$ poetry add watchdog --dev
+```
```
Comment (hugh-emerson) [inline README.md:105]: No, you don't need to install gunicorn on your machine

And you never need to tell people to run `poetry add` anything. Your dependencies are recorded in `pyproject.toml`.

**Review by hugh-emerson — APPROVED** [review]
Looks good! Merge whenever you're ready 👍 

Note that this PR includes ansible files. I've ignored them as Jack reviewed them on your other PR.

File: Dockerfile (hunk 1 above)
Comment (CreaTiVeUK) [reply Dockerfile:25]: Working on a mac. I think the option worked, but I'll note that down. Thanks Hugh.

File: README.md (hunk 2 above)
Comment (CreaTiVeUK) [reply README.md]: Noted and edited. Thanks.

**Pull request merged**